### PR TITLE
add success message if installer succeeds

### PR DESCRIPTION
**Pull request by wojtekCodeInt**

### CODE REVIEW

**Comment by HenrichN** [comment]
I would propose to keep the styling of the success message consistent with our other messages like 
image
image

File: pkg/install/installer.go [hunk 1]
```diff
@@ -373,13 +373,14 @@ func PrintPathInstructions(binDir string) {
...
 		// TODO: On Windows, users generally don't expect having to fiddle with their PATH. We should update it for
 		//       them, but that requires asking for admin access.
-		_, _ = fmt.Fprintf(os.Stderr, `Please add the following directory to your PATH:
+		_, _ = fmt.Fprintf(os.Stderr, `Installation successful. Please add the following directory to your PATH:
     %s
-`, binDir)
+If you haven't already done so.`, binDir)
 	} else {
-		_, _ = fmt.Fprintf(os.Stderr, `Please add the following to ~/.profile or ~/.bash_profile:
+        binDirWithHome := strings.Replace(binDir, "~", "$HOME", 1)
```
Comment (fmeum) [inline pkg/install/installer.go:380]: Can you explain why this replacement is necessary?

Comment (adombeck) [reply pkg/install/installer.go:380]: @wojtekCodeInt ping

**Comment by jochil** [comment]
It would be nice if we can get rid of these "fix go syntax" commits before merging. For the future it can be avoided using this git hooks: https://github.com/CodeIntelligenceTesting/cifuzz/tree/main/githooks

File: pkg/install/installer.go (hunk 1 above)
Comment (jochil) [reply pkg/install/installer.go:380]: what shell are you using?

Comment (wojtekCodeInt) [reply pkg/install/installer.go:380]: GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)

Comment (adombeck) [reply pkg/install/installer.go:380]: >  For me, the commands itself work, but the "which" command does not show their location (just empty output).

I can't reproduce that

Comment (jochil) [reply pkg/install/installer.go:380]: in theory `~` should be a substitution for $HOME (with some extras) and there might be shells not supporting it. But I can reproduce it neither and GNU bash definitely has it. 
So as long we do not know the reason for this behavior, I would prefer going with `~`

Comment (wojtekCodeInt) [reply pkg/install/installer.go]: ok I removed this part.

**Comment by jochil** [comment]
> It would be nice if we can get rid of these "fix go syntax" commits before merging. For the future it can be avoided using this git hooks: https://github.com/CodeIntelligenceTesting/cifuzz/tree/main/githooks

Same for the `remove the ~ to $HOME change as it seems it is not needed that much` commit
@wojtekCodeInt I can squash the commits and force push the changes, if this is ok for you

**Review by jochil — APPROVED** [review]

**Pull request merged**